lib/nimble_template/addons/github.ex

```diff
@@ -51,4 +51,49 @@ defmodule NimbleTemplate.Addons.Github do
 
     project
   end
+
+  @impl true
+  def do_apply(
+        %Project{
+          web_project?: web_project?,
+          mix_project?: mix_project?,
+          erlang_version: erlang_version,
+          elixir_version: elixir_version
+        } = project,
+        opts
+      )
+      when is_map_key(opts, :github_wiki) do
+    binding = [
+      web_project?: web_project?,
+      mix_project?: mix_project?,
+      erlang_version: erlang_version,
+      elixir_version: elixir_version
+    ]
+
+    template_workflow_path = ".github/workflows/publish_wiki.yml.eex"
+    workflow_path = ".github/workflows/publish_wiki.yml"
+
+    template_getting_started_path =
+      if mix_project? do
+        ".github/wiki/Getting-Started.md.mix.eex"
+      else
+        ".github/wiki/Getting-Started.md.eex"
+      end
+
+    getting_started_path = ".github/wiki/Getting-Started.md"
+
+    homepage_path = ".github/wiki/Home.md"
+    sidebar_path = ".github/wiki/_Sidebar.md"
+
+    files = [
+      {:text, template_workflow_path, workflow_path},
+      {:text, homepage_path, homepage_path},
+      {:eex, template_getting_started_path, getting_started_path},
+      {:text, sidebar_path, sidebar_path}
+    ]
```

Comment:
I might prefer putting some variables that are not reused directly.  

```suggestion
    files = [
      {:text, ".github/workflows/publish_wiki.yml.eex", ".github/workflows/publish_wiki.yml"},
      {:text, homepage_path, homepage_path},
      {:eex, template_getting_started_path, ".github/wiki/Getting-Started.md"},
      {:text, sidebar_path, sidebar_path}
    ]
```

I think it's easier to see what is being copied. What do you think? 💭

Reply:
Yeah, I'm fine with this approach too 👍